.github/workflows/check.yml

```diff
@@ -139,3 +153,11 @@ jobs:
           name: pyft_result
           path: |
             neu-ft/neuron/report
+
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v3
+        env:
+          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
Can we try to delete this secret? It doesn't seem friendly enough to the community.

Reply:
It doesn't work if it's deleted.

Reply:
Delete token looks good for me: https://github.com/Rory-Z/neuron/actions/runs/6072392852/job/16472254814